manifest-flags.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
0. This is the preparation step actually for override the values of remote feature flag